params/version.go

```diff
@@ -21,10 +21,10 @@ import (
 )
 
 const (
-	VersionMajor = 1        // Major version component of the current release
-	VersionMinor = 9        // Minor version component of the current release
-	VersionPatch = 8        // Patch version component of the current release
-	VersionMeta  = "stable" // Version metadata to append to the version string
+	VersionMajor = 0  // Major version component of the current release
+	VersionMinor = 9  // Minor version component of the current release
+	VersionPatch = 0  // Patch version component of the current release
+	VersionMeta  = "" // Version metadata to append to the version string
```

Comment:
geth alternates this between `unstable` (not a released version) and `stable` (a released build whose commit corresponds to a tag) -- this would be nicer because otherwise I can't necessarily know this isn't *the* 1.9.0 build

Reply:
Got it, so right now it should be "unstable"